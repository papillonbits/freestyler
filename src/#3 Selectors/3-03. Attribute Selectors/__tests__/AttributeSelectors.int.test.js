import { regular } from './AttributeSelectors.int.story'

describe('<AttributeSelectors />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
