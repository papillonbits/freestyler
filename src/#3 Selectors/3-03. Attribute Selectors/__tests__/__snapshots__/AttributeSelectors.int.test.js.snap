// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AttributeSelectors /> Render must match regular() 1`] = `
<div>
  <div
    class="attribute-selectors"
  >
    <h1>
      Attribute Selectors
    </h1>
    <p>
      This is a paragraph without the 
      <code>
        title
      </code>
       attribute.
    </p>
    <p
      title="Second"
    >
      <code>
        p[title]
      </code>
       selects all paragraphs containing a title attribute.
    </p>
    <p
      title="Third paragraph"
    >
      <code>
        p[title~="paragraph"]
      </code>
       selects all paragraphs with a title attribute containing the word,
       
      <code>
        paragraph
      </code>
      .
    </p>
    <p
      title="#4   paragraph"
    >
      <code>
        p[title="#4 paragraph"]
      </code>
       selects all paragraphs with a title attribute containing the exact text,
       
      <code>
        #4 paragraph
      </code>
      . Matches are case sensitive and must match letter-for-letter including whitespace.
    </p>
    <a
      href="http://www.example.com"
      hreflang="en-GB"
      rel="noreferrer"
      target="_blank"
    >
      hello
    </a>
  </div>
</div>
`;
